### PR TITLE
Test Fix - LRAC-9366 | master

### PR DESCRIPTION
**Pull request by catarineaguiar**
https://github.com/catarineaguiar/liferay-portal/pull/9
https://issues.liferay.com/browse/LRAC-9366

### CODE REVIEW

**Comment by catarineaguiar** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
Hi @catarineaguiar, check why these two tests failed to [run](https://testray.liferay.com/home/-/testray/case_results?testrayBuildId=1563663804) the CI

SegmentsCreation#CanSearchUserInAddMembersModal
SegmentsCreation#CanPreviewDynamicSegmentMember

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase [hunk 1]
```diff
@@ -364,11 +364,15 @@ definition {
...
+			Pause(locator1 = "5000");
+
 			Navigator.gotoSitePage(
 				pageName = "AC Page",
 				siteName = "Site Name");
 
 			DMNavigator.gotoDocumentPG(dmDocumentTitle = "New Document");
+
+			Pause(locator1 = "5000");
 		}
```
Comment (ViniLopes20) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:376]: Hi @catarineaguiar, I think you could put this in `for`

Comment (catarineaguiar) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AssetsDocumentsAndMedia.testcase:376]: Done https://github.com/liferay-qa/liferay-portal/pull/1074

**Pull request closed without merging**